packages/scripts/src/create-entrypoint.ts

```diff
@@ -540,6 +554,10 @@ function fetchFeaturesFromElectronProcess(externalFeaturesRoute: string) {
     return `await require('electron').ipcRenderer.invoke('${externalFeaturesRoute}')`;
 }
 
+function fetchConfigsForElectron(configsRoute: string, envName: string) {
+    return `await require('electron').ipcRenderer.invoke('${LOADED_FEATURE_MODULES_NAMESPACE}.${configsRoute}', { env: '${envName}', featureName, configName })`;
```

Comment:
this basically means that the electron entry (in electron-main target), will add a handler which will return the configuration, instead of the dev-server handling it.